base.css

```diff
@@ -0,0 +1,35 @@
+.navStyle {
+  list-style-type: none;
+  height: 2em;
+  width: 100%;
+}
+
+.link {
+  display: none;
+}
+
+
+@media only screen and (min-width: 640px) {
+  .link {
+    display: inline-block;
+    float: right;
+    padding-left: .5em;
+    padding-right: .5em;
+    margin-right: .5em;
+    margin-bottom: 125px;
+    vertical-align: top;
+  }
+}
+
+@media only screen and (min-width: 640px){
+  #hamIcon {
+    display: none;
+  }
+}
```

Comment:
This second media query looks to be the same as the first, so you would be okay to condense these into one. As a design suggestion, try and see if you can get your links to line up at the top of your header, keeping it even with your name.